target/linux/realtek/files-5.15/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -1740,7 +1739,7 @@ void rtl9300_force_sds_mode(int sds, phy_interface_t phy_if)
 		return;
 	}
 
-	pr_info("%s: SDS mode %x\n", __func__, sds_mode);
+	pr_info("%s --------------------- serdes %d forcing to %x ...\n", __func__, sds, sds_mode);
```

Comment:
Lets move this change into my MR https://github.com/openwrt/openwrt/pull/10453 instead. It keeps this Mr clean; this change has no affect on v5.15 really I think.